pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -462,15 +464,24 @@ func (a AKS) applyOptions(nbv *NodeBootstrapVariables) {
 		return fmt.Sprintf("%s=%s", k, v)
 	}), ",")
 
+	// Assign Per K8s version kubelet flags
+	minorVersion := semver.MustParse(a.KubernetesVersion).Minor
+	if minorVersion < 30 {
```

Comment:
did you ever figure out if we need to touch feature gates as well?

Reply:
I took a look, and it looks like we don't need to here as the defaults of those gates align with what we want to set